ffig/FFIG.py

```diff
@@ -78,14 +78,47 @@ def build_model_from_source(path_to_source, module_name):
     Returns:
     - model built from a clang.cindex TranslationUnit with a name from args
     """
-    tu = clang.cindex.TranslationUnit.from_source(
-        path_to_source, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    try:
+        tu = clang.cindex.TranslationUnit.from_source(
+            path_to_source, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    except TranslationUnitLoadError:
```

Comment:
I agree. For now, we shouldn't worry too much about customising the error handling here. Especially since
- `tu` won't get assigned to if an exception is thrown.
- `print` isn't the right way to handle this in a core library.

Later on, we can build tools around FFIG.py (including the REST API, web site, etc.) to handle error reporting in whatever way makes sense for the application.